src/AppInstallerCLICore/Argument.h

```diff
@@ -49,23 +49,44 @@ namespace AppInstaller::CLI
         // Defines an argument with no alias.
         constexpr static char NoAlias = '\0';
 
+        // Defines an argument with no alternate name
+        constexpr static std::string_view NoAlternateName = "";
+
         Argument(std::string_view name, char alias, Execution::Args::Type execArgType, Resource::StringId desc) :
-            m_name(name), m_alias(alias), m_execArgType(execArgType), m_desc(std::move(desc)) {}
+            m_name(name), m_alias(alias), m_alternateName(NoAlternateName), m_execArgType(execArgType), m_desc(std::move(desc)) {}

```

Comment:
nit: The default constructed state of a `std::string_view` is already an empty string.